modules/identity-provider-htpasswd-secret.adoc

```diff
@@ -3,7 +3,7 @@
 // * authentication/identity_providers/configuring-htpasswd-identity-provider.adoc
 
 [id="identity-provider-creating-htpasswd-secret_{context}"]
-= Creating the HTPasswd Secret
+= Creating the HTPasswd secret
```

Comment:
OT: I've been wondering about this lately, what's the proper English grammar way to write capital letters in headlines?

Reply:
So there are two main ways to write titles: either title case or sentence case.

- For title case, you capitalize every word (except minor ones, like "the", "and", "of", etc.). For example: "Configuring Identity Providers Using the Web Console"
- For sentence case, you write it as you would a normal sentence, only capitalizing words that are proper/need to always be capitalized a certain way (like "OpenShift" or HTPasswd"). For example: "Configuring identity providers using the web console"

In OpenShift docs (at least for 4.x), we use sentence case for our headings.

There's a nice website that lets you input a heading and will convert it properly for either title case or sentence case (among other options): https://titlecase.com/. You'll still have to manually adjust for any words that still need to stay capitalized a certain way though.

Let me know if that answers your question or if you have any others :)

Reply:
Oh, cool, this all is very good to know. I've only known about the title case for English but the sentence casing was brought to my attention recently, so I was curious to get a confirmation :slightly_smiling_face: 

The Czech grammar only acknowledges the sentence form for the titles.

Reply:
That's cool to know. We used to use title case for docs at Red Hat when I started years ago, but over the last few years we've moved to sentence case as a department. I definitely think that sentence case is a lot easier on the eyes :)